.github/auto-release.yml

```diff
@@ -1,53 +0,0 @@
-name-template: 'v$RESOLVED_VERSION'
```

Comment:
why did you delete this?

Reply:
because it was failing and github spammed me with emails. As I do not see any significant reason to have it here, it was deleted. if you want I can get it back and try to fix it.